source/introduction/history-of-ethereum.rst

```diff
@@ -1,3 +1,70 @@
 ********************************************************************************
 History of Ethereum
-********************************************************************************
\ No newline at end of file
+********************************************************************************
+
+Inception
+================================================================================
+Ethereum was initially described by Vitalik Buterin in late 2013 as a result of his research and work in the Bitcoin community. Shortly thereafter, Vitalik published the `Ethereum white paper <http://vbuterin.com/ethereum.html>`_, where he describes in detail the technical design and rationale for the Ethereum protocol and smart contracts architecture. In January 2014, Ethereum was formally announced by Vitalik at the The North American Bitcoin Conference in Miami, Florida, USA. From that moment on, the excitement around Ethereum and the its momentum has grown at a dramatic pace. 
+
+Around that time, Vitalik also started working with Dr. Gavin Wood and together co-founded Ethereum. By April 2014, Gavin published the `Ethereum Yellow Paper <http://gavwood.com/paper.pdf>`_ that would serve as the technical specification for the Ethereum Virtual Machine (EVM). By following the detailed specification in the Yellow Paper, the Ethereum client has been implemented in six programming languages (C++, Go, Python, Java, JavaScript, and Haskell), and has resulted in better software overall.
```

Comment:
Translation? 
1) what is now 7?  Is this a reminder to cross ref client implementation section later? 
2) Gavin and Jeffrey co-authored the yellow paper? 
3) I don't know much more about the foundation and subsidiaries. 
– Is Stiftung Ethereum the same thing as Ethereum Suisse?
– I know that Ethereum Dev is a NFP contractor
– What other orgs?



You insiders know so much more than others.